data_prep/data_utils.py

```diff
@@ -0,0 +1,45 @@
+from data_prep.agnews_data import *
+from data_prep.graph_dataset import *
+from data_prep.imdb_data import *
+from data_prep.pure_graph_dataset import *
+from data_prep.reuters_data import *
+from data_prep.roberta_dataset import *
+from data_prep.roberta_graph_dataset import *
+
+
+def get_dataloaders(model, b_size, data_name):
+    corpus = get_data(data_name)
+    additional_params = {}
+
+    if model == 'roberta':
+        train_loader = RobertaDataset(corpus.train).as_dataloader(b_size, shuffle=True)
+        test_loader = RobertaDataset(corpus.test).as_dataloader(b_size)
+        val_loader = RobertaDataset(corpus.val).as_dataloader(b_size)
+
+        additional_params['num_classes'] = corpus.num_classes
+
+        return train_loader, test_loader, val_loader, additional_params
+
+    if model == 'pure_gnn':
+        dataset = PureGraphDataset(corpus)
+    elif model == 'roberta_gnn':
+        dataset = RobertaGraphDataset(corpus)
+    else:
+        raise ValueError("Model type '%s' is not supported." % model)
+
+    if isinstance(dataset, GraphDataset):
+        additional_params['num_classes'] = corpus.num_classes
+        additional_params['gnn_output_dim'] = dataset.num_nodes
+        train_loader = val_loader = test_loader = dataset.as_dataloader()
+        return train_loader, val_loader, test_loader, additional_params
+
+
+def get_data(data_name):
```

Comment:
Minor: we might want to pass the validation size to this function in case we decide to use something else than 10% of the training data.

Reply:
will incorporate